src/errors.rs

```diff
@@ -67,7 +67,7 @@ quick_error! {
 }
 
 impl cmp::PartialEq for Error {
-    #[allow(match_same_arms)]
+    #[cfg_attr(feature = "cargo-clippy", allow(clippy::match_same_arms))]
```

Comment:
why using `clippy::` here?

Reply:
Clippy suggests it

Reply:
oh, clippy suggests using `clippy` 😭

Reply:
do we have a better way @kennytm

Reply:
@siddontang  The `clippy::` lint namespace is intended to be the proper way to use clippy in the 2018 edition. After the `tool_lints` feature is stabilized (as seen in `src/lib.rs` line 216), you could just write `#[allow(clippy::match_same_arms)]` to let the compiler ignore it outside of clippy.

So I think this is the best way for now.